ddp-workspace/projects/ddp-lms/src/app/pages/how-to-participate/how-to-participate.component.ts

```diff
@@ -7,7 +7,7 @@ import { Component } from '@angular/core';
 })
 export class HowToParticipateComponent {
 
-  consent = 'https://storage.googleapis.com/cmi-study-dev-assets/lms/pdf/7.%20Adult%20Reseach%20Consent%20Form%20V5%20TRACKED%20LMS%20April%202022.docx.pdf';
+  consent = 'https://storage.googleapis.com/cmi-study-dev-assets/lms/pdf/7.%20Adult%20Reseach%20Consent%20Form%20V7%20CLEAN%20LMS%2016Dec%202022.pdf';
```

Comment:
followed whats in there but wondering if prod also is referring to dev-assets !!

Reply:
Added file to git rep under pdfs